crates/lib/src/qpu/quilc/mod.rs

```diff
@@ -14,13 +14,17 @@ use super::{rpcq, Qcs};
 
 mod isa;
 
+/// Number of seconds to wait before timing out.
+pub const DEFAULT_COMPILER_TIMEOUT: u8 = 30;
```

Comment:
```suggestion
pub use super::rpcq::DEFAULT_CLIENT_TIMEOUT;
```'
Or
```suggestion
/// Number of seconds to wait before timing out.
pub const DEFAULT_COMPILER_TIMEOUT: u8 = super::rpcq::DEFAULT_CLIENT_TIMEOUT;
```

Unless there's a specific reason for having these set separately?

Reply:
I know they are the same value, but I see no reason to couple the RPCQ client with the compiler logic. Even if they are used exclusively together at this point.

Reply:
So my understanding is that the compiler timeout is the timeout for the rpcq client when talking with quilc, which would make this one specific use of a client timeout.

If I understand you right, you're wanting to keep them separate because we might want a compiler timeout even if we move away from rpcq in this case, and so setting it to the same value as the rpcq timeout would no longer make sense?

In that case, it makes sense to keep them separate, even if the current use seems duplicative.

Reply:
Exactly, we're on the same page now.